openlibrary/plugins/importapi/code.py

```diff
@@ -348,7 +349,7 @@ def populate_edition_data(self, edition, identifier):
         """
         edition['ocaid'] = identifier
         edition['source_records'] = "ia:" + identifier
-        edition['cover'] = "https://archive.org/download/{0}/{0}/page/title.jpg".format(identifier)
+        edition['cover'] = "{0}/download/{1}/{1}/page/title.jpg".format(IA_BASE_URL, identifier)
```

Comment:
Should this use IA_DOWNLOAD_URL instead of `{0}/download`?

Reply:
This is in a different file / module, and since it is only used once I thought only creating the on URL constant for IA_BASE_URL was sufficient.